windows/d2-installer.nsi

```diff
@@ -355,24 +375,49 @@ Function DetectVSAndSDK
     StrCpy $VCPath $0
     StrCpy $VCVer $1
 
-    ReadRegStr $0 HKLM "Software\Microsoft\Windows Kits\Installed Roots" "KitsRoot81"
+	!insertmacro _DetectSDK "Windows Kits\Installed Roots" "KitsRoot10" "lib\um\x64"
```

Comment:
I have just guessed the folder to check for the Windows 10 SDK. Is it correct or do we have to use the UCRTVersion as the first folder?